korangar/src/loaders/model/mod.rs

```diff
@@ -169,19 +184,21 @@ impl ModelLoader {
     }
 
     fn process_node_mesh(
+        version: InternalVersion,
         current_node: &NodeData,
         nodes: &[NodeData],
         processed_node_indices: &mut [bool],
         vertex_offset: &mut usize,
         native_vertices: &mut Vec<NativeModelVertex>,
         model_texture_mapping: &[ModelTexture],
+        hashmap_texture: &HashMap<String, i32>,
```

Comment:
Can't we use an enum instead? We could then just use one of the texture mappings (vec or map)

```rust
enum TextureMapping<'a> {
    PreVersion2_3(&'a[ModelTexture])
    PostVersion2_3(&'aHashMap<String, i32>)
}
````